docs/design-system.md

```diff
@@ -67,9 +67,17 @@ Icons are separated from components and already exported as `ReactComponent` . Y
 
 ### How to add icons
 
-Icons are separated into sections to make traversing the directory a little easier when you’re unsure of what all is in there. [Material Design icon categories](https://material.io/resources/icons/?style=baseline) were used as the naming convention so as not to reinvent the wheel.
+Icons are all located directly in the `design-system/icons/` folder. A few things to note:
 
-1. Export your SVG & compress it (example: [https://jakearchibald.github.io/svgomg/](https://jakearchibald.github.io/svgomg/))
-2. Add the file to the appropriate directory in [design-system/icons](../assets/src/design-system/icons/index.js)
-3. Export it from the index of that directory as a ReactComponent
+- All icons are 32x32px and should never be resized or scaled in any way.
+- Icons are named after their symbol, not their function. So a ⚙️  icon is named `Gear`, and not `Settings` or some other function, it might serve.
+- If an icon has several sizes, denote this as a suffix as `GearLarge`, `GearTiny` etc.
+- If an icon has a border, name it either just as `GearOutline` or as the shape of the border - e.g. `GearTriangle` if it's a gear inside a triangular border.
+- Try to group similar icons alphabetically, so name icons e.g. `MagnifierPlus` and `MagnifierMinus` rather than `PlusMagnifier` and `MinusMagnifier`.
+
+The process for adding an icon is as follows:
+
+1. Export your SVG from Figma by selecting the 32x32px icon boundary box and selecting _"Export"_ in the right panel
+2. Add the file to directory [design-system/icons](../assets/src/design-system/icons/index.js)
+3. Export it from the index of that directory as a ReactComponent (please alphabetize the list, thanks!)
```

Comment:
Should we add a step here to compress the svg first?

Reply:
optimization is happening by default in webpack - https://github.com/google/web-stories-wp/pull/6287#discussion_r573089953

Reply:
That supposed to happen automagically in the build pipeline, but I'm actually not sure if we still do this.

@swissspidy, can you confirm that this happens? I only see `svgr` in the [Webpack config](https://github.com/google/web-stories-wp/blob/main/webpack.config.cjs#L95), but no mention of `svgo`.

[This doc](https://react-svgr.com/docs/options/#svgo) indicates that `svgo` is standard when using `svgr`, so I assume it works?

Reply:
It should still be the default, yes. There's also a `.svgo.yml` config file for it that - at least when I last tested it - was applied to SVGs.

You could quickly test this yourself by removing those configs and re-building the app. It should cause the icons in the app to appear rather broken :-).

If it doesn't, then there's a bug.